### ISSUE TITLE
Cursor misalignment

### ISSUE THREAD

**Issue by jhakonen** [ubuntu/yaru#1423]
The normal cursor's point isn't at mouse's coordinates. It seems to be a few pixels too high and a bit to the left. See attached screenshot:
![misaligned-normal-cursor](https://user-images.githubusercontent.com/1950698/60827985-09af8e80-a1ba-11e9-9e90-de0f573f4714.png)

In the picture I have changed settings from Universal Access --> Zoom:
* Set zoom to 2.00
* Enabled cross hair and made it as thin as possible 

It seems that other cursor types have similar issues as well.

Using Ubuntu 19.04 with the Yaru theme that came with it.

In the mean time I've switched to Hackneyed cursor theme (https://gitlab.com/Enthymeme/hackneyed-x11-cursors) as it seems to be better aligned. Here's for a comparison what the position probably should look like:
![aligned-hackneyed-cursor](https://user-images.githubusercontent.com/1950698/60828753-e71e7500-a1bb-11e9-94fc-d09cb29fa9ad.png)

```
$ snap info communitheme
name:      communitheme
summary:   The next Ubuntu theme built by the community.
publisher: Didier Roche (didrocks)
license:   GPL-3.0
description: |
  Yaru, formerly known as Communitheme, is the new Ubuntu theme built by the
  community. Yaru will become the default Ubuntu theme in Ubuntu 18.10. This
  package allows you to try out the theme on Ubuntu 18.04 LTS.
  
  To try out the theme, install this package on Ubuntu 18.04 LTS, restart
  your computer and select the "Ubuntu with communitheme snap" session from
  the login screen.
  
  More information is available at
  https://community.ubuntu.com/t/faq-ubuntu-new-theme/1930.
snap-id: Yd6CISPIf6tEf3ZEJ0cqSoEg9rG2VkRi
channels:
  stable:    0.1 2019-03-13 (1768) 16MB -
  candidate: ↑                          
  beta:      ↑                          
  edge:      0.1 2019-07-06 (1823) 17MB -
```

**Comment by fitojb** [comment]
Indeed, if one puts the mouse pointer all the way to the screen’s left edge, one can notice the missing border.

**Comment by Amr-Ibra** [comment]
I wish the cursor could be sharpened more.

**Comment by rustushki** [comment]
I can confirm that this is still an issue. I was investigating a mouse cursor intersection issue in another application. After a lot of debugging, I found that the Yaru mouse pointer was a couple of pixel two low relative to the visible tip of the cursor image. The Adwaita cursor which is similar in appearance does not have this problem.

**Comment by ubuntujaggers** [comment]
I think I know how to fix this 👍

**Comment by Feichtmeier** [comment]
Isn't this fixed? @madsrh @ubuntujaggers ?

**Comment by Feichtmeier** [comment]
Fixed with #1825
